Extensions/TileMap/JsExtension.js

```diff
@@ -526,9 +526,13 @@ module.exports = {
 
         // Check if the point is inside the object bounds
         const originalWidth =
-          this._pixiTileMapData?.tilemapWidth / this._pixiObject.scale.x;
+          (this._pixiTileMapData
```

Comment:
Same, don't use the width or the height of the pixi object.

Don't try to use something because something else is not available. If you've decided that the pixi object size is not the proper one, then never use it.

`const originalWidth = this._pixiTileMapData ? (this._pixiTileMapData.tilemapWidth / this._pixiObject.scale.x) : 0;`

Reply:
what if its been resized? I guess we still need to do something to differentiate between the customSize tilemap and the original size tilemap